app/src/sharedTest/java/org/oppia/app/topic/play/TopicPlayFragmentTest.kt

```diff
@@ -131,6 +140,90 @@ class TopicPlayFragmentTest {
     intended(hasExtra(StoryActivity.STORY_ACTIVITY_STORY_ID_ARGUMENT_KEY, storyId))
   }
 
+  @Test
+  fun testTopicPlayFragment_loadFragmentWithTopicTestId0_chapterListIsNotVisible() {
+    activityTestRule.launchActivity(null)
+    onView(atPositionOnView(R.id.story_summary_recycler_view, 0, R.id.chapter_recycler_view)).check(
+      matches(
+        not(
+          isDisplayed()
+        )
+      )
+    )
+  }
+
+  @Test
+  fun testTopicPlayFragment_loadFragmentWithTopicTestId0_clickExpandListIcon_iconChanges() {
```

Comment:
This is testing two behaviors (one case where the arrow is down, and a second where the arrow is up). Please split into two tests to be focused for each behavior being tested.

Reply:
Added another test case 
`testTopicPlayFragment_loadFragmentWithTopicTestId0_clickExpandListIconIndex1_clickExpandListIconIndex0_chapterListForIndex0IsNotDisplayed`